### DISCUSSION TITLE
Add ability to manage multiple ER diagrams within the same model

### DISCUSSION THREAD

**Discussion by michael-rozet** [pgmodeler/pgmodeler#1929] [Ideas]
Is there a way to have multiple ER diagrams within the same model? Basically I'm looking for a feature allowing to:
1. display a subset of tables on a diagram
2. have multiple diagrams each focusing on a subset of tables (aka Subject Areas)

**Comment by rkhaotix** [comment]
Hi @michael-rozet 

Maybe the implementation of the layers will help you?

See here:
https://pgmodeler.io/support/docs/615-layers-management?v=1.2.0

**Comment by michael-rozet** [comment]
@rkhaotix the layers do help, up to a point - thanks. The main issue with this approach is dependency between layers: if a table is included in more than one layer, its placement will be common to all of them. E.g., there is no way to have two diagrams with the same set of tables arranged differently.

**Reply by rkhaotix** [reply]
It's already on my plans to add support for objects' multiple positions depending on the layers they are :)

**Comment by sggreener** [comment]
An observation. If one groups elements via schemas, or layers, it might be nice to be able to select objects based on these groups and then use that selection in a DDL export.
